swift_browser_ui_frontend/src/components/ObjectTable.vue

```diff
@@ -113,6 +113,18 @@
       id="objects-toasts"
       data-testid="objects-toasts"
     />
+    <c-toasts
+      id="largeDownload-toasts"
+      data-testid="largeDownload-toasts"
+    >
+      <p>{{ this.$t("message.largeDownMessage") }}</p>
```

Comment:
As github indicates, no need for `this.` here. `$t` is injected by the plugin, and is available in the templates.

Reply:
Oops! Thanks for spotting, fixed